datadog_cluster_agent/datadog_checks/datadog_cluster_agent/check.py

```diff
@@ -54,6 +54,12 @@
     'rate_limit_queries_remaining_min': 'datadog.rate_limit_queries.remaining_min',
     'rate_limit_queries_reset': 'datadog.rate_limit_queries.reset',
     'secret_backend__elapsed_ms': 'secret_backend.elapsed',
+    'language_detection_dca_handler_ok_response': 'language_detection_dca_handler.ok_response',
+    'language_detection_dca_handler_fail_response': 'language_detection_dca_handler.fail_response',
+    'language_detection_patcher_retries': 'language_detection_patcher.retries',
+    'language_detection_patcher_success_patch': 'language_detection_patcher.success_patch',
+    'language_detection_patcher_fail_patch': 'language_detection_patcher.fail_patch',
+    'language_detection_patcher_skipped_patch': 'language_detection_patcher.skipped_patch',
```

Comment:
It looks like all the previous `DEFAULT_METRICS` lines were alphabetically ordered.
Could this block be sorted and inserted line 51 ?